### PR TITLE
[hotfix-api-error] Anticipate possible `null` API response.

### PR DESCRIPTION
**Pull request by jgerigmeyer**
https://oddbird.slack.com/archives/CB2576XT4/p1557426417013800

armadillo

### CODE REVIEW

File: src/js/store/jobs/actions.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const startJob = (data: JobData): ThunkAction => (
   })
     .then(response => {
       /* istanbul ignore else */
-      if (window.socket) {
+      if (response && window.socket) {
```
Comment (davisagli) [inline src/js/store/jobs/actions.js]: If we got this far we're going to dispatch JOB_STARTED which is not actually the case in the scenario we've been talking about, right?

Comment (jgerigmeyer) [reply src/js/store/jobs/actions.js]: That is a very good point.

File: src/js/store/jobs/actions.js [hunk 2]
```diff
@@ -114,8 +114,11 @@ export const startJob = (data: JobData): ThunkAction => (
     },
   })
     .then(response => {
+      if (!response) {
+        throw new Error('Invalid response received');
+      }
```
Comment (davisagli) [inline src/js/store/jobs/actions.js]: will this be caught and shown to the user? just go into the console?

Comment (jgerigmeyer) [reply src/js/store/jobs/actions.js]: Just the console -- at least until https://trello.com/c/DmfdVJCB

**Pull request merged**